install.sh

```diff
@@ -6,5 +6,8 @@ git submodule init
 git submodule sync
 git submodule update
 
-# for react-native
-npm install react-native@0.50.4 react@16.0.0 --silent
+
+# get react native
+pushd "libs/test/SalesforceReactTest"
```

Comment:
We don't want it at the top level otherwise the react packager gets confused by the javascript code for cordova.